src/main/java/com/onepagecrm/net/request/Request.java

```diff
@@ -71,6 +74,7 @@
     protected static final String NETWORK_DEV_NAME = "NETWORK";
     protected static final String CUSTOM_NAME = "CUSTOM";
 
+    protected static final String AUTH_URL = "http://auth.mse.onepagecrm.eu/api/v3/"; // TODO: update URL before going live
```

Comment:
@CillianMyles  This need to be updated?

Reply:
@lorobode This can be merged into `mse/main` but needs to be updated before included in a build which is published on mobile!